backend/src/graphql/resolvers/UserResolver.ts

```diff
@@ -86,6 +88,17 @@ export class UserResolver {
     if (!result.success) {
       throw new Error(result.data)
     }
+    const qluser = new User(result.data.user)
```

Comment:
I suggest `user` here as it is imported.

Reply:
Okay, I exchange user with dbuser and qluser with user.